data/json/items/gun/460sw.json

```diff
@@ -4,7 +4,7 @@
     "copy-from": "pistol_revolver",
     "looks_like": "sw_619",
     "type": "GUN",
-    "name": { "str_sp": "S&W 460XVR" },
+    "name": { "str_sp": ".460 S&W Magnum revolver" },
```

Comment:
Could you add the non-generic name as a variant? If you don't want to, that's fine, I can do it

Reply:
I shouldn't have forgotten to do that. Thank you.